src/finalfusion/embeddings.py

```diff
@@ -246,7 +246,10 @@ def embedding_with_norm(self,
                 out[:] = default[0]
                 return out, default[1]
             return default
-        return self._embedding(idx, out)
+        # declare the norm as Any, self._embedding returns Optional[float] but since line 244
```

Comment:
You probably don't want to put a line number here. Once another part changes, it's out-of-sync.

Reply:
Funnily enough, it's already out of sync. fixed!